src/test/java/FizzBuzzTest.java

```diff
@@ -41,24 +42,24 @@ private void doFizzBuzz(int n, String s) throws IOException {
 		System.out.flush();
 		assertEquals(s, baos.toString());
 	}
+	
+	private String generateFizzBuzzTestStringsUntil20 (int length) {
+		if (length < 1 || length > 20) {
+			throw new IllegalArgumentException("generateFizzBuzzTestStringsUntil20 must be called with a length from 1-20");
+		}
+		String[] outputs = {"1", "2", "Fizz", "4", "Buzz", "Fizz", "7", "8", "Fizz", "Buzz", "11", "Fizz", "13", "14", "FizzBuzz", "16", "17", "Fizz", "19", "Buzz"};
+		String output = "";
```

Comment:
This will need to be rewritten using a `StringBuilder` to conform to our coding style guides. String concatenation in loops is disallowed.